rclcpp/test/test_logging.cpp

```diff
@@ -163,6 +164,38 @@ TEST_F(TestLoggingMacros, test_logging_skipfirst) {
   }
 }
 
+TEST_F(TestLoggingMacros, test_throttle) {
+  using namespace std::chrono_literals;
+  for (uint64_t i = 0; i < 3; ++i) {
+    RCLCPP_DEBUG_THROTTLE(g_logger, rcutils_steady_time_now, 10000, "Throttling");
+  }
+  EXPECT_EQ(1u, g_log_calls);
+  RCLCPP_DEBUG_SKIPFIRST_THROTTLE(g_logger, rcutils_steady_time_now, 1, "Skip first throttling");
```

Comment:
@BMarchi hmm, having to pass an `rcutils` function to an `rclcpp` logging macro is somewhat odd. Consider taking `rclcpp::Clock` instances instead and filling the gap between layers.